src/cpu/matmul/cpu_matmul_list.cpp

```diff
@@ -43,7 +43,7 @@ using namespace dnnl::impl::data_type;
 using namespace dnnl::impl::cpu::matmul;
 
 // clang-format off
-const impl_list_item_t impl_list[] = REG_MATMUL_P({
+constexpr impl_list_item_t impl_list[] = REG_MATMUL_P({
```

Comment:
There are other lists that could be made `constexpr`, is there a reason why you changed only these two?

Reply:
Because the other ones, as far as I could tell, do not produce load-time content.

Reply:
Can you mark the remaining lists as `constexpr` for consistency?